uncrustify.cfg

```diff
@@ -1,15 +1,17 @@
 set FOR foreach
 set WORD slots
 
-newlines			= LF
+newlines				= LF
 
 indent_with_tabs		= 0
 indent_columns			= 4
+indent_namespace		= true
+indent_class			= true
+indent_col1_comment		= true
+indent_access_spec		= -4
 indent_switch_case		= 0
-indent_namespace        	= true
-indent_class            	= true
-indent_col1_comment     	= true
-indent_access_spec      	= -4
+indent_case_shift		= 0
+indent_case_brace		= 4
 
 nl_after_func_body		= 2
 nl_collapse_empty_body	= false
```

Comment:
@evsh:  This line does not seem to be spaced using a tab.

Reply:
I have no idea why some of the lines are shown differently in GitHub diff viewer. If you look in the code viewer (I mean the "<> Code" tab) you will see the correct formatting. And everything is aligned with tabs in my local copy (the new file).

Reply:
@evsh: 

>If you look in the code viewer (I mean the "<> Code" tab) you will see the correct formatting. 

Already tried.  In the code-viewer, I see the **exact same** alignment as the diff-viewer.